kebechet/managers/update/messages.py

```diff
@@ -156,8 +156,7 @@
 
 Make sure your `Pipfile` or `requirements.in` or `requirements-dev.in` is placed in the root of your Git repository.
 
-/kind feature
-/priority important-soon
+/kind feature /priority important-soon
```

Comment:
this needs to be on two different lines or they will all be considered /kind I think